.goreleaser.yml

```diff
@@ -69,8 +69,8 @@ dockers:
   build_flag_templates:
   - "--pull"
   - "--build-arg=kan_tools_version={{ .Tag }}"
-# Refers to https://github.com/kopia/kopia/commit/5c85ca6bb087781eb3ea4044ffd87f0c00530c0a
-  - "--build-arg=kopia_build_commit=5c85ca6"
+# Refers to https://github.com/kopia/kopia/commit/02463ab118c391c520995cc907dc070e944054cb
+  - "--build-arg=kopia_build_commit=02463ab"
```

Comment:
kopia/kopia@1e98511c2ea07fbcfbe4cf0f29b58900bb32e460 ?

Reply:
Latest commit updated in https://github.com/kanisterio/kanister/pull/2848